doc/release-notes.md

```diff
@@ -31,197 +30,279 @@ Notable changes
+
+### Other security-related fixes
+
+- Harden the protocol implementation to reject buggy or malformed messages.
+  This helps protect the network against broken custom clients.
+- Limit and tightly manage memory usage in events of high network traffic or
+  when connected to extremely slow peers. This protects nodes on lower end
+  hardware to not run out of memory in the face of increased network activity.
+
+Fee Recommendation
 ------------------
 
-The Berkley DB version used by Dogecoin Core has been updated to 5.3 (from 5.1)
-as 5.3 is now standard on many Linux distributions. 5.1 and 5.3 wallet files
-have been tested to be interchangeable.
+This release changes the recommended dust limit for all participants on the
+Dogecoin network from 1 DOGE to 0.01 DOGE. The full recommendation can be found
+[in the documentation](fee-recommendation.md).
 
-Version display
----------------
+This change has been implemented in the wallet as the default value of
+`-discardthreshold`, resulting in the following default logic:
 
-The version displayed on QT's overview page has been changed to display the
-full version rather than just the major version part, because this was confusing
-wallet users.
+1. The wallet will attempt to not create any outputs smaller than 0.03 DOGE as
+   change if possible.
+2. If after signing there is more than 0.01 DOGE left in the change output, the
+   wallet will keep the output as-is.
+3. If the change output drops under 0.01 DOGE, the wallet will discard the
+   change and instead give it to miners as fee.
+4. Any output under 0.01 DOGE will not be accepted as valid by default.
 
-Key Derivation
---------------
+Breaking changes
+----------------
 
-The BIP32 hierarchical deterministic key derivation path contained the wrong
-chain ID. Previously the chain ID 0 was used, it's now correctly set to 3 as
-per [SLIP44](https://github.com/satoshilabs/slips/blob/master/slip-0044.md).
+### Configuration of a wallet backup directory
 
-The wallet.dat files stay fully interoperable between versions. Wallets created
-with 1.14.5 will benefit from greater interoperability with hardware wallets in
-the future.
+Adds a new command line parameter `-backupdir` that allows operators to specify
+where to store wallet backups and dumps. When not set, the node will add a
+directory under the data directory called "backups".
 
-Namecoin-compatibile AuxPoW mining
-----------------------------------
+**This is a breaking change.**
 
-The `createauxblock` and `submitauxblock` commands have been reintroduced,
-mimicking the same commands from Namecoin 0.17, allowing miners to separate
-wallets from block producing nodes by specifying the address for their coinbase
-transactions.
+The behavior of the `dumpwallet` and `backupwallet` RPC methods has changed,
+as these will ignore any path given other than the filename, and always store
+the filename in the directory defined with `-backupdir` or the above-mentioned
+default location.
 
```

Comment:
Adding a note here to remind myself (or anyone who does it before I do) to document that these commands will also refuse to overwrite existing files.